types/types.go

```diff
@@ -17,7 +17,10 @@ const (
 	EngineBinaryDirectoryInContainer = "/engine-binaries/"
 	EngineBinaryDirectoryOnHost      = "/var/lib/rancher/longhorn/engine-binaries/"
 
-	LonghornNodeKey = "longhornnode"
+	LonghornLabelKey = "longhorn"
```

Comment:
Let's use `nodes.longhorn.io/role=storage`. We will convert to use a more standard label format.